lib/main.js

```diff
@@ -18,43 +18,54 @@ function HTMLUglify(config) {
 
   var salt = config.salt || 'use the force harry';
```

Comment:
It does? https://github.com/RebelMail/html-preflight/blob/master/lib/plugins/uglifier.js#L6